lib/src/ConfigLoader.cc

```diff
@@ -532,6 +532,7 @@ static void loadDbClients(const Json::Value &dbClients)
             characterSet = client.get("client_encoding", "").asString();
         }
         auto timeout = client.get("timeout", -1.0).asDouble();
+        auto keepalive = client.get("keepalive_", -1.0).asDouble();
```

Comment:
Should be `client.get("keepalive", -1.0)` ?

Reply:
My mistake I fixed it.